src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,8 @@
 package mate.academy.lib;
 
-public @interface Component {
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
 
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Component {
```

Comment:
Add https://github.com/target here

Reply:
@Ivan95kos What do you mean? 
Should I add this here @Target(ElementType.TYPE) or what?